src/nebula-hooks/use-sorting.ts

```diff
@@ -4,8 +4,17 @@ import { Column, HyperCube } from '../types';
 export const sortingFactory = (model: EngineAPI.IGenericObject | undefined, hyperCube: HyperCube) => {
   if (!model) return undefined;
 
-  return async (column: Column) => {
+  return async (column: Column, sortDirection?: string) => {
```

Comment:
In which scenario is `sortDirection` falsy and what does that mean?

Reply:
when you press the header, you don't pass sortDirection. you only pass it when you click the menu